File: 71A - Way Too Long Words/main.php
<?php
/*
    https://codeforces.com/problemset/submission/71/225065457
    PHP 8.1.7 - 46 ms - 24 KB
*/

$file = fopen('php://stdin', 'r');
$size = fgets($file);

for($i = 0; $i < $size; $i++) {
    $s = trim(fgets($file));
    if(strlen($s) > 10) {
        echo($s[0] . strlen($s) - 2 . $s[strlen($s) - 1] . "\n");
    } else {
        echo($s . "\n");
    }
}

fclose(($file));
?>

File: 71A - Way Too Long Words/main.c
/*
    https://codeforces.com/problemset/submission/71/225063483
    GNU C11 - 15 ms - 0 KB
*/

#include<stdio.h>
#include <string.h>
#define MAX 500

int main(int argc, char const *argv[])
{
    char string[MAX];
    int size;
    scanf("%d", &size);
    getchar();
    for(int i=0; i<size; i++) {
        fgets(string, MAX, stdin);
        string[strcspn(string, "\n")] = 0; // remove \n in string
        if (strlen(string) > 10) {
            printf("%c%d%c\n",string[0], strlen(string)-2, string[strlen(string)-1]);
        } else {
            printf("%s\n", string);
        }
    }
    return 0;
}
